cli/medperf/commands/compatibility_test/utils.py

```diff
@@ -48,11 +48,13 @@ def prepare_local_cube(path):
     config.tmp_paths.append(dst)
     cube_metadata_file = os.path.join(path, config.cube_metadata_filename)
     if not os.path.exists(cube_metadata_file):
+        mlcube_yaml_path = os.path.join(path, config.cube_filename)
+        mlcube_yaml_hash = get_file_hash(mlcube_yaml_path)
         temp_metadata = {
             "id": None,
             "name": temp_uid,
             "git_mlcube_url": "mock_url",
-            "mlcube_hash": "",
+            "mlcube_hash": mlcube_yaml_hash,
             "parameters_hash": "",
```

Comment:
Are we checking only mlcube.yaml hash? What about parameters?

Reply:
@VukW Only `mlcube.yaml`.

Reply:
It's because `git_mlcube_url` is required, and we set it to `mock_url`. Others are optional, so we don't provide URLs, and hence no hashes are required/checked.